src/main.js

```diff
@@ -44,6 +44,8 @@ Vue.mixin({
 	},
 })
 
+Vue.prototype.$OC = OC
```

Comment:
We don't need that one.

Reply:
If I remove it, I get: Error in mounted hook: "TypeError: this.$OC is undefined"

Reply:
Should be fine once the assignment is gone :wink: